lang/en.yml

```diff
@@ -2,7 +2,7 @@ en:
   SilverStripe\RealMe\Authenticator\LoginForm:
     ASSERTLOGINBUTTON: 'Share your details with {orgname}'
     AUTHENTICATOR_NAME: 'RealMe Account'
-    LOGINBUTTON: Login
+    LOGINBUTTON: Log in
```

Comment:
We have a policy of not updating existing sources en.yml in because it deletes translations from translations. Instead you should create LOGINBUTTON2 and use that instead.

Reply:
Doesn't matter in this case but done anyway